lib/workos/profile.rb

```diff
@@ -13,9 +13,9 @@ class Profile
     extend T::Sig
 
     sig { returns(String) }
-    attr_accessor :id, :email, :first_name, :last_name,
+    attr_accessor :id, :email, :first_name, :last_name, :connection_id,
                   :connection_type, :idp_id, :raw_attributes
-
+    # rubocop:disable Metrics/AbcSize
```

Comment:
@willmanduffy @marktran How do we feel about this rubocop configuration change instead of pulling assignments out into `before_action`?

Reply:
https://github.com/workos-inc/workos-ruby/pull/60#discussion_r505659519